src/operator/swapaxis-inl.h

```diff
@@ -0,0 +1,197 @@
+/*!
+ * Copyright (c) 2015 by Contributors
+ * \file swapaxis-inl.h
+ * \brief
+ * \author Ming Zhang
+*/
+#ifndef MXNET_OPERATOR_SWAPAXIS_INL_H_
+#define MXNET_OPERATOR_SWAPAXIS_INL_H_
+
+#include <dmlc/logging.h>
+#include <dmlc/parameter.h>
+#include <mxnet/operator.h>
+#include <algorithm>
+#include <map>
+#include <vector>
+#include <string>
+#include <utility>
+#include "./operator_common.h"
+
+namespace mxnet {
+namespace op {
+
+struct SwapAxis{
```

Comment:
Actually change to name space wilI need small captital letter
On Sun, Nov 8, 2015 at 10:11 PM Zhang Ming <notifications@github.com> wrote:

> In src/operator/swapaxis-inl.h
> <https://github.com/dmlc/mxnet/pull/519#discussion_r44243657>:
>
> > +#define MXNET_OPERATOR_SWAPAXIS_INL_H_
> > +
> > +#include <dmlc/logging.h>
> > +#include <dmlc/parameter.h>
> > +#include <mxnet/operator.h>
> > +#include <algorithm>
> > +#include <map>
> > +#include <vector>
> > +#include <string>
> > +#include <utility>
> > +#include "./operator_common.h"
> > +
> > +namespace mxnet {
> > +namespace op {
> > +
> > +struct SwapAxis{
>
> what do you mean?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dmlc/mxnet/pull/519/files#r44243657>.
>